### PR TITLE
Addings project workflows steps

### PR DESCRIPTION
**Pull request by Vinetos**
Adding basics commands for the project workflow

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -6,3 +6,18 @@ Depandencies :
 
 gcc
 make
+
+Prepare the project :
+
+```shell
```
Comment (Vinetos) [inline README]: This a text file, no need this

File: README [hunk 2]
```diff
@@ -6,3 +6,18 @@ Depandencies :
...
+git clone git@github.com:C-Text/CText.git
+git flow init
+```
+
+Create a new feature :
+```shell
```
Comment (Vinetos) [inline README]: Same thing here

File: README (hunk 1 above)
Comment (Vinetos) [reply README]:
```suggestion

```

**Review by starcruiser5289 — COMMENTED** [review]
noice job out there @Vinetos

**Review by starcruiser5289 — APPROVED** [review]
take n*2

**Pull request merged**